packages/taquito-sapling/src/types.ts

```diff
@@ -23,11 +36,129 @@ export interface Ciphertext {
   nonceOut: Buffer;
 }
 
-export interface SaplingTransaction {
-  spendDescriptions: SpendDescription[];
-  outputDescriptions: OutputDescription[];
+export interface SaplingTransactionInput {
+  cv: Buffer;
+  nf: Buffer;
+  rk: Buffer;
+  proof: Buffer;
```

Comment:
is there documentation for these to know what the abbreviation is referring to?

Reply:
There is, but would it be better if I use the whole name instead of the abbreviation?

Reply:
i feel like for people working that know what it is it would be fine to keep as is but if people are new it would have to be fairly clear what they are supposed to be so if there is documentation for it I'm sure it would be fine

Reply:
Documentation is a bit spread, I renamed the variables to be more descriptive